optaplanner-examples/src/main/resources/org/optaplanner/examples/machinereassignment/machineReassignmentSolverConfig.xml

```diff
@@ -16,9 +16,6 @@
       <!--<incrementalScoreCalculatorClass>org.optaplanner.examples.machinereassignment.optional.score.MachineReassignmentIncrementalScoreCalculator</incrementalScoreCalculatorClass>-->
     <!--</assertionScoreDirectorFactory>-->
   </scoreDirectorFactory>
-  <termination>
-    <minutesSpentLimit>5</minutesSpentLimit>
-  </termination>
```

Comment:
As agreed, please do not change the default solver configs.
Ruin should be introduced in a separate benchmark config first.